zhmccli/_cmd_ldap.py

```diff
@@ -0,0 +1,362 @@
+# Copyright 2023 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Commands for HMC LDAP server definition management.
+"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import click
+
+import zhmcclient
+from .zhmccli import cli
+from ._helper import print_properties, print_resources, abort_if_false, \
+    options_to_properties, original_options, COMMAND_OPTIONS_METAVAR, \
+    click_exception, add_options, LIST_OPTIONS
+
+
+def find_ldap_srv_def(cmd_ctx, console, ldap_srv_def_name):
```

Comment:
minor: why abbreviate srv? Makes it just harder to read and pronounce.

Reply:
The normal name would be `ldap_server_definition` which is 22 charecters long. The naming rules defined in our pylint config file have max lengthts of names of 28..32 depending on the kind of name. I think that 22 is too close to that and limits the ability to express the meaning of the variable. Therefore, I went for an abbreviation. How about `ldapsd`?

Reply:
I have changed:
* any internal names from `ldap_srv_def` to `ldapsd`,
* the metavar of the LDAP Server Definition name in the help text from `LDAP_SRV_DEF` to `LDAP_SERVER_DEFINITION`
* the module name from "ldap.py" to "ldap_server_definition.py"